app/views/account/users/_form.html.slim

```diff
@@ -0,0 +1,10 @@
+= simple_form_for [:account, @user],
+        defaults: {input_html: {class: 'form-control'}} do |f|
+  = f.input :first_name, autofocus: true
+  = f.input :last_name
+  = f.input :email
+  = f.input :role , collection: [User::ADMIN_ROLE,User::SALE_ROLE], include_blank: false
```

Comment:
could ya add some class method on user model to make it easier to use. mb like:
```
collection: User.roles_for_select
# where
# User.roles_for_select #=> [User::ADMIN_ROLE,User::SALE_ROLE]
```

Reply:
or User::ROLES = [ADMIN_ROLE, SALE_ROLE...]
collection: User::ROLES